8.0/root/usr/libexec/container-setup

```diff
@@ -41,18 +41,19 @@ mkdir -p ${APP_ROOT}/etc
 chmod -R a+rwx ${APP_ROOT}/etc
 chmod -R a+rwx ${HTTPD_VAR_RUN}
 chown -R 1001:0 ${APP_ROOT}
-mkdir /tmp/sessions
+mkdir -p /tmp/sessions
 chmod -R a+rwx /tmp/sessions
 chown -R 1001:0 /tmp/sessions
 chown -R 1001:0 ${HTTPD_DATA_PATH}
 chmod -R a+rwx ${PHP_SYSCONF_PATH}
-mkdir -p /run/php-fpm
-chmod -R a+rwx /run/php-fpm
-chown -R 1001:0 /run/php-fpm
-mkdir -p /var/log/php-fpm
-chmod -R a+rwx /var/log
-#chown -R 1001:0 /var/log/php-fpm
-
+if [ "x$PLATFORM" == "xel9" ]; then
```

Comment:
Could be better to use same test (`[ -d "/run/php-fpm" ]`) everywhere for consistency ?

Reply:
notice: especially as php-fpm can also be used (if wanted) on EL-8

Reply:
Do we want to support PHP-8.0 also for RHEL8 containers? I would prefer to strictly use it only in CentOS Stream 9.

Reply:
Addressed by https://github.com/sclorg/s2i-php-container/pull/347/commits/a839d56b3aa873bd226fea6082b0f2d0eeee26fc. As condition for CentOS Stream 9 as check for existence directory `/run/php-fpm` is present.

Reply:
The question is if we would like to add PHP-FPM also in Dockerfile.rhel8, which is now present, or we would like to use only `mod_php` like in `7.4/Dockerfile.rhel8` https://github.com/sclorg/s2i-php-container/blob/master/7.4/root/usr/share/container-scripts/php/common.sh#L29. Nowadays I would prefer for Dockerfile.rhel8 use mod_php. Like before.